common/test/acceptance/pages/studio/users.py

```diff
@@ -254,6 +254,7 @@ def click_delete(self):
         self.wait_for_element_visibility('.prompt', 'Prompt is visible')
         self.wait_for_element_visibility('.prompt .action-primary', 'Confirmation button is visible')
         self.q(css='.prompt .action-primary').click()
+        self.wait_for_element_absence('.page-prompt .is-shown', 'Confirmation prompt is hidden')
```

Comment:
Note for reviewers: the code after this method was sometimes getting a "wrong page" error when trying to verify something on the page-- I believe that was because the modal dialog was still visible. The impacted test passed 30 times in Jenkins with this change.